src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -1,25 +1,62 @@
-# Overview 
+# Azure App Configuration client library for .NET
+Azure App Configuration is a managed service that helps developers centralize their application configurations simply and securely.
 
-Azure.ApplicationModel.Configuration is a component of the .NET Azure SDK. 
-It provides APIs for Microsoft [Azure's Application Configuration Service](https://docs.microsoft.com/en-us/azure/azure-app-configuration/) which is a service that allows to easily store and manage all application settings in one central place that is separated from code.
+Modern programs, especially programs running in a cloud, generally have many components that are distributed in nature. Spreading configuration settings across these components can lead to hard-to-troubleshoot errors during an application deployment. Use App Configuration to store all the settings for your application and secure their accesses in one place.
```

Comment:
s/generally/often?

Reply:
"secure their accesses" is awkward phrasing to my ears. You could maybe say "Use App Configuration to securely store all the settings for your application in one place".